packages/plugin-git/sources/hosted-git-info-parse.ts

```diff
@@ -33,7 +33,7 @@ function safeURL(url: string) {
 // attempt to correct an scp style url so that it will parse with `new URL()`
 function correctURL(gitURL: string) {
   // ignore @ that come after the first hash since the denotes the start
-  // of a committish which can contain @ characters
```

Comment:
[committish](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish) is correct here